Tests/Microsoft.AppCenter.Test.UWP/Microsoft.AppCenter.Test.UWP.csproj

```diff
@@ -11,13 +11,14 @@
     <AssemblyName>Microsoft.AppCenter.Test.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```

Comment:
Please revert this file, this is not part of the version bump script and should not be touched during a release.